brax/physics/config_pb2.py

```diff
@@ -1,20 +1,6 @@
-# Copyright 2021 The Brax Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: brax/physics/config.proto
```

Comment:
Thanks for regenerating these!  I realize we don't have instructions for contributors on how to do this - I'll add them in the future.  The only issue here is that this path information is important, as it propagates down into the message descriptor properties.  So to regenerate, instead of doing it within `brax/physics/`, instead do:

```cd <my-brax-project-dir>
python3 -m grpc_tools.protoc -I . --python_out=.  brax/physics/config.proto
```